registers/generated-emulator/src/mci.rs

```diff
@@ -48,7 +48,7 @@ pub trait MciPeripheral {
         &mut self,
     ) -> caliptra_emu_bus::ReadWriteRegister<u32, registers_generated::mci::bits::HwRevId::Register>
     {
-        caliptra_emu_bus::ReadWriteRegister::new(0)
+        caliptra_emu_bus::ReadWriteRegister::new(0x1000)
```

Comment:
This is a generated file. Is there a better mechanism to update this value?

Reply:
The better way is to overwrite this method in the peripheral that implements this trait.